rosdep/python.yaml

```diff
@@ -2017,6 +2017,20 @@ python-gnupg:
   rhel:
     '7': [python2-gnupg]
   ubuntu: [python-gnupg]
+python-gobject:
+  arch: [python2-gobject]
+  debian:
+    '*': null
+    buster: [python-gobject]
+    stretch: [python-gobject]
+  fedora: [python3-gobject]
```

Comment:
isn't this the python3 package?
It looks like you want to add the python2 version here, right?

Reply:
@ivanpauno yes, you're right. removed fedra key because it does not provide python2 binding and fix nixos .